wagtail/contrib/wagtailsitemaps/sitemap_generator.py

```diff
@@ -9,7 +9,7 @@ def __init__(self, site=None):
         self.site = site
 
     def location(self, obj):
-        return obj.full_url
+        return obj.url
```

Comment:
@gasman correct, which is why I changed it back to `obj.url`. According to the [django documentation](https://docs.djangoproject.com/en/1.11/ref/contrib/sitemaps/#django.contrib.sitemaps.Sitemap.location) `location` should return an absolute path without the domain.
The only change being made here is dropping the `.specific` since `items` now returns specific page objects.